docs/tau-updates.md

```diff
@@ -0,0 +1,128 @@
+# Targetability Updates (TAU)
+
+In cases where fflogs sends insufficient information to determine the correct information and state of a boss (invulnerability window, ephemeral enemies, etc), there is a mechanism to override this on a per-fight basis under `src/parser/bosses`.
```

Comment:
Hmmmm... semi-accurate. More precise; the mechanism is part of the `invuln` module, `src/parser/bosses` is just the part of the meta system that loads the invuln override. Same as overriding core modules for job logic.